Corona-Warn-App/src/main/res/values-de/covid_certificate_strings.xml

```diff
@@ -57,6 +57,8 @@
     <string name="vaccination_list_immunity_card_body">"Sie haben nun alle derzeit geplanten Impfungen erhalten. Ihr Impfschutz ist vollständig."</string>
     <!-- XTXT: Vaccination card title -->
     <string name="vaccination_state_title">"Impfstatus"</string>
+    <!-- XTXT: Vaccination card booster eligible -->
+    <string name="vaccination_card_booster_eligible">"Aufgrund Ihres vorhanden Impfzertifikats könnte für Sie eine Auffrischungsimpfung erforderlich sein, da Ihr letztes Impfzertifikat älter als 6 Monate ist (Regel-ID: %1$s)."</string>
```

Comment:
```suggestion
    <string name="vaccination_card_booster_eligible">"Aufgrund Ihres vorhandenen Impfzertifikats könnte für Sie eine Auffrischungsimpfung erforderlich sein, da Ihr letztes Impfzertifikat älter als 6 Monate ist (Regel-ID: %1$s)."</string>
```

Reply:
@BMItr  why did you add it to the strings? is it required to translate it to all languages? I think we can just get the description from the rule itself for EN, DE right?

Reply:
done